snapshots/default.json

```diff
@@ -12,13 +12,13 @@
     "commit": "c1bad34"
   },
   "bufferline.nvim": {
-    "commit": "ef4fdc5"
+    "commit": "243893b"
```

Comment:
For information, this fixes the following error in Neovim v0.10.0-dev:

```
E5108: Error executing lua ...site/pack/lazy/opt/bufferline.nvim/lua/bufferline/ui.lua:132: Segments must be a list
stack traceback:
        [C]: in function 'assert'
        ...site/pack/lazy/opt/bufferline.nvim/lua/bufferline/ui.lua:132: in function 'get_component_size'
        ...site/pack/lazy/opt/bufferline.nvim/lua/bufferline/ui.lua:504: in function <...site/pack/lazy/opt/bufferline.nvim/lua/bufferline/ui.lua:485>
        vim/shared.lua: in function 'get_components'
        ...im/site/pack/lazy/opt/bufferline.nvim/lua/bufferline.lua:85: in function <...im/site/pack/lazy/opt/bufferline.nvim/lua/bufferline.lua:81>
```

Refs.
- https://github.com/akinsho/bufferline.nvim/issues/727
- https://github.com/akinsho/bufferline.nvim/pull/721